pkg/app_watcher/app_watcher_test.go

```diff
@@ -8,12 +8,18 @@ import (
 	"github.com/argoproj/argo-cd/v2/pkg/apis/application/v1alpha1"
 	appclientsetfake "github.com/argoproj/argo-cd/v2/pkg/client/clientset/versioned/fake"
 	"github.com/stretchr/testify/assert"
+	"github.com/zapier/kubechecks/pkg/config"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
 	"github.com/zapier/kubechecks/pkg/appdir"
 )
 
-func initTestObjects() *ApplicationWatcher {
+func initTestObjects(t *testing.T) *ApplicationWatcher {
+	cfg, err := config.New()
+	if err != nil {
+		// Handle the error appropriately, e.g., log it or fail the test
+		t.Fatalf("Failed to initialize config: %v", err)
+	}
```

Comment:
`require.NoError(err)` would be a little more succinct.

Reply:
changed to 
assert.NoError(t, err, "failed to create config")